src/main/java/org/Terasology/Volcanoes/VolcanoProvider.java

```diff
@@ -29,6 +29,7 @@
 })
 @Produces(VolcanoFacet.class)
 public class VolcanoProvider implements FacetProviderPlugin {
+    private static final int DENSITY_SAMPLING_FACTOR = 0.07;

```

Comment:
Thinking you meant `float` here? Easy fix while merging, tests fine as `float` 👍